scene/gui/code_edit.cpp

```diff
@@ -1601,6 +1609,27 @@ bool CodeEdit::can_fold_line(int p_line) const {
 		return false;
 	}
 
+	/* Check for code region. */
```

Comment:
```suggestion
	// Check for code region.
```
Use single line comments for these cases